src/util/std_expr.h

```diff
@@ -203,14 +203,26 @@ inline symbol_exprt &to_symbol_expr(exprt &expr)
 class nondet_symbol_exprt : public nullary_exprt
 {
 public:
-  /// \param identifier: Name of symbol
-  /// \param type: Type of symbol
+  DEPRECATED(SINCE(
+    28, 05, 2019,
+    "Use nondet_symbol_exprt(irep_idt, typet, source_locationt) instead"))
```

Comment:
I disagree. There is really nothing wrong with this constructor. No need to deprecate it.